src/include/snapper/dialogs.rb

```diff
@@ -556,62 +563,95 @@ def SummaryDialog
             update_snapshots.call
             next
           end
+
         elsif ret == :create
           if CreateSnapshotPopup(pre_snapshots)
             update_snapshots.call
             next
           end
+
         elsif ret == :modify
           if ModifySnapshotPopup(Ops.get(snapshots, selected, {}))
             update_snapshots.call
             next
           end
+
         elsif ret == :delete
           if DeleteSnapshotPopup(Ops.get(snapshots, selected, {}))
             update_snapshots.call
             next
           end
+
         elsif ret == :next
           break
+
         else
           Builtins.y2error("unexpected retcode: %1", ret)
           next
         end
+
       end
 
       deep_copy(ret)
     end
 
+
+    def generate_ui_file_tree(subtree)
+      return subtree.children.map do |file|
```

Comment:
It's not needed for the interpreter but helps humans to understand the code, e.g. whether the function returns a useful value or just by accident the value of the last statement. And after all we write code for others to read.